docs/src/scripts/generate_llms.py

```
@@ -0,0 +1,151 @@
+    tabs = config["navigation"]["languages"][0]["tabs"]
+    for tab in tabs:
+        out += ["", f"# {tab['tab']}", ""]
+        for group in tab.get("groups", []):
+            out += [f"## {group['group']}", ""]
+            out += render_pages(group.get("pages", []), depth=3)
+            out += [""]
+        if "openapi" in tab:
+            try:
+                spec = fetch_openapi(tab["openapi"])
+                out += render_openapi(spec)
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)